sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -74,8 +81,8 @@ def log_prob(
 
         Args:
             theta: Parameters $\theta$.
-            x: Conditioning context for posterior $p(\theta|x)$. Per default, use the
-               observation self.x_o.
+            x: Conditioning context for posterior $p(\theta|x)$. Can be set for doing
+                amortized inference. Per default, use the observation self.x_o.
```

Comment:
"Can be passed here explicitly to evaluate the posterior given different $x$, e.g., in case of amortized inference. Per default, `self.x_o` as specified during initialization is used. "